ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -212,8 +212,11 @@ param set IMU_INTEG_RATE 250
 . px4-rc.params
 
 dataman start
+# start sih in sih_sim mode, otherwise simulator module
+if [ "$SIM_MODE" = "sih_sim" ]; then
```

Comment:
Should we respect `SYS_HITL`?

Reply:
I would say no, cause we are not in HITL any more, we are in SITL.
To do it cleanly we should rename it SYS_SIM (0 for real flight, 1 for HITL, 2 for SIH_HITL, and 3 for SIH_SITL for instance).

Reply:
Yes I just mean to respect the parameter centrally rather than depend on environment variables.

Reply:
Here, I'd say `SYS_HITL` should be 0, as we run in SITL. Adding a new environment variable is not ideal here. We just need some way of starting `sih` and not `simulator` module here. I don't think the `SYS_HITL` param helps here?